src/region/actions.ts

```diff
@@ -1,16 +1,14 @@
 import { AppThunkDispatch, AppState } from '../store';
 import { createAnnotationAction } from '../store/annotations';
 import { getFileVersionId } from '../store/options';
-import { Rect } from '../@types';
+import { AnnotationRegion } from '../@types';
 
-export type CreateArg = {
-    location: number;
+export interface CreateArg extends Pick<AnnotationRegion, 'target'> {
```

Comment:
I don't think this is a good idea. The purpose of these abstractions was to separate the "create" argument from the annotation types. Can we revert these changes, please?